src/components/playlisteditor/playlisteditor.ts

```diff
@@ -99,6 +113,29 @@ function redirectToPlaylist(id: string | undefined) {
     appRouter.showItem(id, currentServerId);
 }
 
+function updatePlaylist(dlg: DialogElement) {
+    const apiClient = ServerConnections.getApiClient(currentServerId);
+    const api = toApi(apiClient);
+
+    if (!dlg.playlistId) return Promise.reject(new Error('Missing playlist ID'));
```

Comment:
I'd move this to the beginning of the function to avoid unnecessary calls in case of an error.